ironfish/src/memPool/feeEstimator.ts

```diff
@@ -141,6 +141,19 @@ export class FeeEstimator {
     return result
   }
 
+  estimateFeeRates(): { priority: PriorityLevel; feeRate: bigint }[] {
```

Comment:
```suggestion
  /* returns an estimated fee rate as ore/kb */
  estimateFeeRates(): { priority: PriorityLevel; feeRate: bigint }[] {
```

Reply:
After this merges.. what do you think about just keying this? Otherwise users are gonna have to iterate the array to get the level they want. Especially since its a finite set.

Reply:
I think keying is a better approach. I hadn't really thought about getting a particular level, only about getting all of them.

Since this is new functionality I think I'll make the change here before merging.